<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>650</width>
      <height>761</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QVTKOpenGLStereoWidget" name="openGLWidget_All">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>630</width>
       <height>400</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QVTKOpenGLStereoWidget" name="openGLWidget_Main">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>310</width>
       <height>340</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QVTKOpenGLStereoWidget" name="openGLWidget_Sub">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>420</y>
       <width>310</width>
       <height>340</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QProgressBar" name="PanoProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="CaptureReady">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>촬영준비</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CaptureStartPushButton">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>50</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>촬영시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>50</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>촬영정지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>촬영종료</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>229</y>
      <width>320</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ceph RAW</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pano RAW(90°)</string>
     </property>
    </widget>
    <widget class="QLabel" name="PanoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>150</width>
       <height>500</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CephLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>150</width>
       <height>500</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>영상 데이터 전송</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>90</y>
      <width>261</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <row>
     <property name="text">
      <string>Raw 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Raw 10</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>통신 로그</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>저장소</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>환자 번호</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>날짜</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>장비정지</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>장비시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>장비종료</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AscendingPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>상승</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DescendingPushButton">
     <property name="geometry">
      <rect>
       <x>87</x>
       <y>60</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>하강</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MainPushButton">
     <property name="geometry">
      <rect>
       <x>163</x>
       <y>60</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>메인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SubPushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>서브</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="PanoCheckBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pano Ready</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CephCheckBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ceph Ready</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLStereoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglstereowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
